### ISSUE TITLE
Bank change MSB LSB and other MIDI stuff

### ISSUE THREAD

**Issue by cooptrol** [polyend/TrackerBetaTesting#1419]
_The PT needs to have Bank MSB and LSB options plus the PCH message. I know there is only 2 effects per step but some hardware can’t respond solely to MSB and PCH to change patches. We need more CCs, a way to send Bank MSB + LSB + PCH per track if there is not going to be more than 2 effects per step.

_Most importantly: Program change when switching patterns!!! Some hardware doesn't handle well a PCH each time the pattern loops, so it would be great to have a PCH message sent ONLY when you switch the pattern (like most pattern-driven sequencers). 

_To have MIDI out to be also THRU, or more precisely, MERGE, so you can connect a MIDI controller to the in and use knobs to control the external hardware connected to the OUT. And it would be ace to be able to input CC values via MIDI in and get them to be recorded on to the steps too while repeated to OUT. 

_CCs in the Performance mode effects. 

Hope there is a great os 1.6.0 in the oven!!!

**Comment by eturnigy** [comment]
i agree, i would allso like to trigger patterns from midi in as i slave my polyend tracker
(with program change command)
